cli/internal/cmd/iamdestroy.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/cli/internal/cloudcmd"
+	"github.com/edgelesssys/constellation/v2/internal/constants"
+	"github.com/edgelesssys/constellation/v2/internal/file"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func runDestroyIAMUser(cmd *cobra.Command, _args []string) error {
+	spinner := newSpinner(cmd.ErrOrStderr())
```

Comment:
```suggestion
	spinner := newSpinner(cmd.ErrOrStderr())
	defer spinner.Stop()
```
I'm not familiar with the spinner, but from other code I see that it seems to need a deferred call to Stop()